runtime/src/genesis_utils.rs

```diff
@@ -1,3 +1,4 @@
+//use solana_programs::get_default_native_instruction_processors;
```

Comment:
nit: remove

Reply:
removed in https://github.com/solana-labs/solana/pull/5333